teacher.rb

```diff
@@ -0,0 +1,7 @@
+require './person'
+
+class Teacher < Person
+  def can_use_services?
+    true
+  end
+end
```

Comment:
- [ ] let's add function initialize where Constructor extends parent's constructar by adding @specialization and a parameter for it. Link to the example about constructors in a previous lesson https://github.com/microverseinc/curriulum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars
 
```
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization

```